streamer/aws_products_config.yaml

```diff
@@ -0,0 +1,152 @@
+products:
+  wofs_albers:
+    prefix: WOfS/WOFLs/v2.1.5/combined
+    name_template: x_{x}/y_{y}/{time:%Y}/{time:%m}/{time:%d}/LS_WATER_3577_{x}_{y}_{time:%Y%m%d%H%M%S%f}
+    stacked_name_template: x_{x}/y_{y}/{time:%Y}/{time:%m}/{time:%d}/LS_WATER_3577_{x}_{y}_{time:%Y}_
+    predictor: 2
+    nonpym_list: ["source", "observed"]
```

Comment:
`nonpym_list` contains the key words of the band names which one doesn't want to generate overviews. You might want to be careful about this 'cause it will match as `*source*`. For most products, this term is not needed. So far, only fractional cover percentile use this.

Reply:
Updated config file to reflect `nonpym_list` only to `fc_percentile` products.